src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center center;
+    margin-right: 4px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
```

Comment:
Follow the checklist, please 
![image](https://github.com/mate-academy/layout_product-cards/assets/99403198/00d79db5-d0f4-4c26-b84b-45755e144bc1)